src/store/cache/useCacheStore.ts

```diff
@@ -69,13 +74,64 @@ const store: StateCreator<CacheStore> = (set, get) => ({
 	...initialState,
 	initCache: async (projectId, force = false) => {
 		await Promise.all([
+			get().fetchResources(projectId, force),
 			get().fetchDeployments(projectId, force),
 			get().fetchTriggers(projectId, force),
 			get().fetchVariables(projectId, force),
 			get().fetchConnections(projectId, force),
 		]);
 	},
 
+	fetchResources: async (projectId, force) => {
+		const dbService = new IndexedDBService("ProjectDB", "resources");
+
+		const resources = await dbService.getAll();
+		const { currentProjectId } = get();
+
+		if (currentProjectId === projectId && !force) {
+			return resources;
+		}
+
+		set((state) => ({
+			...state,
+			loading: { ...state.loading, resourses: true },
+		}));
+		try {
+			const { data, error } = await ProjectsService.getResources(projectId);
+
+			if (error) {
+				return;
+			}
+			await dbService.clearStore();
+			for (const [name, content] of Object.entries(data || {})) {
+				await dbService.put(name, new Uint8Array(content));
+			}
+			const resources = await dbService.getAll();
```

Comment:
Let's reduce this db query:
```suggestion
			const resources = data;
```

Reply:
refactored